media-gfx/freecad/freecad-0.16.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils fortran-2 python-single-r1 fdo-mime
+
+DESCRIPTION="QT based Computer Aided Design application"
+HOMEPAGE="http://www.freecadweb.org/"
+SRC_URI="https://github.com/FreeCAD/FreeCAD/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+#sci-libs/orocos_kdl waiting for Bug 604130 (keyword ~x86)
+#dev-qt/qtgui:4[-egl] and dev-qt/qtopengl:4[-egl] : Bug 564978 
+COMMON_DEPEND="dev-cpp/eigen:3
+	dev-libs/boost[python]
+	dev-libs/xerces-c[icu]
+	dev-python/matplotlib
+	dev-python/pyside[X]
+	dev-python/shiboken
+	dev-qt/designer:4
+	dev-qt/qtgui:4[-egl]
+	dev-qt/qtopengl:4[-egl]
+	dev-qt/qtsvg:4
+	dev-qt/qtwebkit:4
+	media-libs/coin
+	|| ( sci-libs/opencascade:6.9.1[vtk] sci-libs/opencascade:6.9.0[vtk] sci-libs/opencascade:6.8.0 sci-libs/opencascade:6.7.1 )
+	sys-libs/zlib
+	virtual/glu
+	media-libs/freetype
+	dev-java/xerces
+	${PYTHON_DEPS}"
+RDEPEND="${COMMON_DEPEND}
+	dev-qt/assistant:4
+	dev-python/pivy
+	dev-python/numpy"
+DEPEND="${COMMON_DEPEND}
+	dev-python/pyside-tools
+	>=dev-lang/swig-2.0.4-r1:0"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.14.3702-install-paths.patch
+)
+
+# https://bugs.gentoo.org/show_bug.cgi?id=352435
+# https://www.gentoo.org/foundation/en/minutes/2011/20110220_trustees.meeting_log.txt
+RESTRICT="mirror"
+
+# TODO:
+#   DEPEND and RDEPEND:
+#		salome-smesh - science overlay
+#		zipio++ - not in portage yet
+
+S="${WORKDIR}/FreeCAD-${PV}"
+
+DOCS=(README.md ChangeLog.txt)
+
+pkg_setup() {
+	fortran-2_pkg_setup
+	python-single-r1_pkg_setup
+
+	[[ -z "${CASROOT}" ]] && die "empty \$CASROOT, run eselect opencascade set or define otherwise"
+}
+
+src_configure() {
+	export QT_SELECT=4
+
+#-DOCC_* defined with cMake/FindOpenCasCade.cmake
+#-DCOIN3D_* defined with cMake/FindCoin3D.cmake
+#-DSOQT_ not used
+#-DFREECAD_USE_EXTERNAL_KDL="ON" waiting for Bug 604130 (keyword ~x86)
+	local mycmakeargs=(
+		-DOCC_INCLUDE_DIR="${CASROOT}"/inc
+		-DOCC_LIBRARY_DIR="${CASROOT}"/$(get_libdir)
+		-DCMAKE_INSTALL_PREFIX="${EPREFIX}"/usr/$(get_libdir)/${P}
+		-DCMAKE_INSTALL_DATADIR=share/${P}/
+		-DCMAKE_INSTALL_DOCDIR=share/doc/${P}/
```

Comment:
needs to be `-DCMAKE_INSTALL_DOCDIR=share/doc/${PF}/`, soz for not saying this the last time (https://dev.gentoo.org/~ulm/pms/head/pms.html check the `--docdir` flag)